docs/feature-gates.md

```diff
@@ -96,6 +97,17 @@ which is closer to its origin with this feature. Refer to this
 [link](https://kubernetes.io/docs/concepts/services-networking/topology-aware-hints/)
 for more information.
 
+
+### ProxyTerminatingEndpoints
+
+`ProxyTerminatingEndpoints` enables ProxyTerminatingEndpoints support in AntreaProxy. 
+The feature ProxyTerminatingEndpoints is at beta stage in Kubernetes 1.22 (beta), and
+it is enabled by default in Kubernetes 1.26. For AntreaProxy, traffic can be routed 
+terminating Endpoints in an effort to improve the traffic engineering capabilities. 
```

Comment:
I do not understand this sentence. Consider rephrasing. Actually, I feel we can remove it, and let reader refer to the K8s link to learn more.

Reply:
Sure, I'll remove it. This is the document link https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1669-proxy-terminating-endpoints

Reply:
Updated, could you have a look at it? @jianjuns  Thanks.